Project-II-Calculator/calculator/src/App.css

```diff
@@ -0,0 +1,8 @@
+.calculator-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 40%;
```

Comment:
Be careful, your calculator is deforming on large screen. I believe in this case would be better to use px as units.